tools/profiling/memory/memory_diff.py

```diff
@@ -175,20 +179,37 @@ def _run():
                     text += "{}: {}\n".format(key, cur[key])
                 else:
                     text += "{}: {} -> {}\n".format(key, old[key], cur[key])
+                    diff = cur[key] - old[key]
                     if "call" in key:
-                        call_diff_size += cur[key] - old[key]
+                        if diff < -call_significant:
+                            call_decrease += 1
+                        elif diff > call_significant:
+                            call_increase += 1
                     else:
-                        channel_diff_size += cur[key] - old[key]
-
-    print("CALL_DIFF_SIZE: %f" % call_diff_size)
-    print("CHANNEL_DIFF_SIZE: %f" % channel_diff_size)
-    check_on_pr.label_increase_decrease_on_pr(
-        "per-call-memory", call_diff_size, 64
-    )
-    check_on_pr.label_increase_decrease_on_pr(
-        "per-channel-memory", channel_diff_size, 1000
-    )
-    # TODO(chennancy)Change significant value when minstack also runs for channel
+                        if diff < -channel_significant:
+                            channel_decrease += 1
+                        elif diff > channel_significant:
+                            channel_increase += 1
+
+    print("CALL: %d increases, %d decreases" % (call_increase, call_decrease))
+    print("CHANNEL: %d increases, %d decreases" % (channel_increase, channel_decrease))
+    # if anything increased ==> label it an increase
+    # otherwise if anything decreased (and nothing increased) ==> label it a decrease
+    # otherwise label it neutral
+    # -- this biases reporting towards increases, which is what we want to act on
```

Comment:
Agreed, I think this is the right information to show on a PR today.

Unrelated to this PR: this data would also be great to surface in other ways, for instance if 3/4 of the channel benchmarks show a significant decrease by a large margin, and 1/4 shows a small but significant increase ... that's good data for a handful of reasons.

Reply:
Yeah... would be great to record this stuff to some kind of database somewhere... it's infrastructure work I can't take on right now though